integration-tests/cli/state-import.it.js

```diff
@@ -0,0 +1,131 @@
+import { oneLine } from 'common-tags'
+import StateImport from '@commercetools/state-importer'
+import { getCredentials } from '@commercetools/get-credentials'
+import { clearData } from './helpers/utils'
+import states from '../../packages/state-importer/test/helpers/sampleStates.json'
+
+let projectKey
+if (process.env.CI === 'true') projectKey = 'state-import-integration-test'
```

Comment:
This feels a bit dirty I think. It there another way to configure this? Can't the test run pass the arg?

Reply:
Passing it as args at run time has creates 2 issues:
1. This would imply using one project for all integration tests. As the tests are run in parallel, race conditions could occur (for example: `product-json-to-csv` still running and `product-exporter` begins its teardown phase, deleting all existing products in the project). A way around this will be to run the `--runInBand` flag on CI but this will drastically slow down the int tests

2. Some of the tests are not properly structured and do not properly setup/teardown as should. So even using the `--runInBand` as described above will cause some tests to fail due to unexpected data.

It should be noted that this was the former approach, and we only changed to use a separate project for each test file due to the issues above (See #267)